src/add-to-project.ts

```diff
@@ -1,5 +1,6 @@
 import * as core from '@actions/core'
 import * as github from '@actions/github'
+import * as check from './check-custom-fields'
```

Comment:
Let's rename this module to something more generic, like `utils`.

```suggestion
import * as utils from './utils'
```